content/overview/pricing/whats-included.md

```diff
@@ -2,11 +2,14 @@
 menu:
   overview:
     parent: pricing
-title: What’s included
+title: What’s included?
 weight: 10
 aliases:
   - /docs/intro/pricing/whats-included
   - /intro/pricing/whats-included
+  - /docs/intro/pricing/paying-for-otherservices
```

Comment:
a couple of things:
there's three new `paying-for-otherservices` urls. do we want to add a third `whats-included` (`/overview/pricing/paying-for-other-services`)
also two of these have no hyphen between `otherservices` and there's `other-services`. i think having the hyphen between each word helps to be consistent